Rewritten autoinstallation chapter of client configuration guide (#917)
uyuni-project/uyuni-docs: modules/client-configuration/pages/autoinst-cdrom.adoc

[[autoinst-cdrom]]
= Autoinstallation via a CD-ROM

For clients that are not yet registered to {productname}, and if PXE booting is not an option, a bootable CD-ROM can be used to install the system.

> One option to prepare such a CD-ROM is to use Cobbler.

Reviewer: Replace prepare with finish. THIS IS A TEST.

Author: THIS IS A TEST. it works

Reviewer: Original version accepted.

For information about using Cobbler to prepare a CD-ROM, see xref:client-configuration:autoinst-cdrom.adoc#build-iso-with-cobbler[Build an ISO Image With Cobbler].

The other option is to use mechanisms specific to the distribution:
* For {suse} systems, prepare a CD-ROM using Kiwi.

>   For information, see xref:client-configuration:autoinst-cdrom.adoc#build-iso-with-kiwi[Build a SUSE ISO Image With Kiwi].

Reviewer 2: ```suggestion
  For more information, see xref:client-configuration:autoinst-cdrom.adoc#build-iso-with-kiwi[].
```

* For {redhat} systems, use ``mkisofs``.

>   For information, see xref:client-configuration:autoinst-cdrom.adoc#build-iso-with-mkisofs[Build a RedHat ISO Image With mkisofs].

Reviewer 2: ```suggestion
  For more information, see xref:client-configuration:autoinst-cdrom.adoc#build-iso-with-mkisofs[].
```
